build/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -vx
```

Comment:
Is this something we wish to do?
```
   -v        Print shell input lines as they are read.
   -x        Print commands and their arguments as they are executed.

```

Reply:
Just a `--verbose` doing an equivalent `set`?

Reply:
I failed to notice -vx.  I don't think we want that in our build script.   You can use -verbose to see the actual compile commands, no need for more verbosity.